data-pipeline.yml

```diff
@@ -0,0 +1,222 @@
+
+      features:
+        taskclusterProxy:
+          true
+
+    metadata:
+      name: bugbug train defect model
+      description: bugbug train defect model
+      owner: release-mgmt-analysis@mozilla.com
+      source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
+
+  - ID: train-regression
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '1 hour'}
+    provisionerId: aws-provisioner-v1
+    workerType: tutorial
+    dependencies:
+      - commit-retrieval
+      - bugs-retrieval
+    scopes:
+      - docker-worker:cache:services-testing-bugbug-data
+    payload:
+      maxRunTime: 86400
+      image: mozilla/bugbug-train-regression
+
+      artifacts:
+        public/regressionmodel.xz:
+          path: /regressionmodel.xz
+          type: file
+
+      cache:
+        services-testing-bugbug-data: /cache
+
+      features:
+        taskclusterProxy:
+          true
+
+    metadata:
+      name: bugbug train regression model
+      description: bugbug train regression model
+      owner: release-mgmt-analysis@mozilla.com
+      source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
+
+  - ID: train-tracking
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '1 hour'}
+    provisionerId: aws-provisioner-v1
+    workerType: tutorial
+    dependencies:
+      - commit-retrieval
+      - bugs-retrieval
+    scopes:
+      - docker-worker:cache:services-testing-bugbug-data
+    payload:
+      maxRunTime: 86400
+      image: mozilla/bugbug-train-tracking
+
+      artifacts:
+        public/trackingmodel.xz:
+          path: /trackingmodel.xz
+          type: file
+
+      cache:
+        services-testing-bugbug-data: /cache
+
+      features:
+        taskclusterProxy:
+          true
+
+    metadata:
+      name: bugbug train tracking model
+      description: bugbug train tracking model
+      owner: release-mgmt-analysis@mozilla.com
+      source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
+
+  - created: {$fromNow: ''}
+    deadline: {$fromNow: '1 hour'}
+    provisionerId: aws-provisioner-v1
+    workerType: releng-svc
+    dependencies:
+      - train-tracking
+      - train-regression
+      - train-defect
+      - train-component
+    payload:
+      capabilities:
+        privileged: true
+      maxRunTime: 3600
+      image: mozilla/taskboot:latest
+      command:
+        - "/bin/sh"
+        - "-lcxe"
+        - "git clone ${repository} /code &&
+           cd /code &&
+           git checkout ${head_rev} &&
+           python http_service/download_models.py &&
+           taskboot --target /code build-compose --registry=registry.hub.docker.com --write /images --service bugbug-http-service"
+      artifacts:
+        public/bugbug:
+          expires: {$fromNow: '2 weeks'}
+          path: /images
+          type: directory
+    scopes:
+      - docker-worker:capability:privileged
+    metadata:
+      name: bugbug docker http build
```

Comment:
Will we push the image to AWS/Heroku in the same task, or in a task dependent on this one?

Reply:
A dependent one